packages/replayio/src/utils/recordings/types.ts

```diff
@@ -37,6 +37,27 @@ export type LogEntry = {
   recordingId?: string;
   server?: string;
   timestamp: number;
+  baseURL?: string;
+  targetContentHash?: string;
+  targetURLHash?: string;
+  targetMapURLHash?: string;
+  parentId?: string;
+  parentOffset?: number;
+};
+
+export type OriginalSource = {
+  path: string;
+  parentOffset: number;
+};
+
+export type SourceMap = {
+  id: string;
+  path: string;
+  baseURL: string;
+  targetContentHash?: string;
+  targetURLHash?: string;
+  targetMapURLHash: string;
+  originalSources: OriginalSource[];
```

Comment:
nit: I think it helps with merge conflicts to always alpha-sort object/type attributes (unless there's some reason _not_ to, e.g. `...rest`)